### ISSUE TITLE
Results from DrMemory tool not taken in account

### ISSUE THREAD

**Issue by Tom3592** [SonarOpenCommunity/sonar-cxx#1796]
### Description

The execution of DrMemory tool is ok and the output result file contain the right information.
The sonar-scanner tool shows this message :
"DEBUG: 'C++ (Community) DrMemorySensor' skipped because there is no related rule activated in the quality profile" 

Finally, there is no issue in SonarQube from DrMemory.

### Steps to reproduce the problem

Please provide the steps required to reproduce the problem

1. Write and compile a dummy C++ code

```
#include <iostream>

int main(int argc, const char* argv[]) {

    std::cout << "Hello World !" << std::endl;
    int* ptr = new int [10000]; // hello valgrind or DrMemory !
    return 0;
}
```
Compilation:

`$ g++ -o test -g HelloWorld.cpp`

2. Run DrMemory and sonar-scanner

Notes: 
* I'am using SonarQube 7.9.2 in a Docker container.
* All these operations run well with valgrind (memory link is detected => major issues)

sonar-project-properties file:

```
sonar.cxx.drmemory.reportPath=/home/user/HelloWorld/logs/**/results.txt
```

```
$ drmemory -logdir /home/user/HelloWorld/logs -- /home/user/HelloWorld/test
$ sonar-scanner -Dsonar.host.url=http://localhost:9000 -X
```

### Expected behavior

Message in sonar-scanner showing that DrMemorySensor is activated

### Actual behavior

The sonar-scanner tool shows this message :
"DEBUG: 'C++ (Community) DrMemorySensor' skipped because there is no related rule activated in the quality profile" 

Finally, there is no issue in SonarQube from DrMemory.

### Known workarounds

no workaround (except using valgrind).
Which rules to put in SonarQube in a given Quality Profile ?
Should an administrator write custom rules for DrMemory  (Brrr) ?

### LOG file

```
$  sonar-scanner -Dsonar.host.url=http://localhost:9000 -X | grep -i memory
12:56:26.134 DEBUG: 'logs/symcache/libdrmemorylib.so.txt' indexed with language 'null'
12:56:26.138 DEBUG: 'logs/DrMemory-test.12423.000/global.12423.log' indexed with language 'null'
12:56:26.138 DEBUG: 'logs/DrMemory-test.12423.000/results.txt' indexed with language 'null'
12:56:26.139 DEBUG: 'logs/DrMemory-test.12423.000/missing_symbols.txt' indexed with language 'null'
12:56:26.139 DEBUG: 'logs/DrMemory-test.12423.000/suppress.txt' indexed with language 'null'
12:56:26.140 DEBUG: 'logs/DrMemory-test.12423.000/potential_errors.txt' indexed with language 'null'
12:56:26.365 DEBUG: 'C++ (Community) DrMemorySensor' skipped because there is no related rule activated in the quality profile
12:56:26.738 INFO: Final Memory: 6M/24M
```

### Related information

* cxx plugin version: 1.3.1
* SonarQube version; 7.9.2
* Sonar-scanner: 4.2.0.1873 (but same results with an old version 3.0.3)

**Comment by guwirth** [comment]
Hi @Tom3592 ,

[Ensure that a rule is enabled](
https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Activate-quality-rules) if you get no results. In new SQ versions the default profile is read-only. The cxx plugin does not enable rules for external tools per default.

Regards,

**Comment by guwirth** [comment]
@Tom3592 please reopen if still an issue.
